filter-assingment/Dhanashri/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta chaet="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Filter Assignment </title>
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+    
+
+<body>
+
+    <section>
+        <div class="container">
+
+            <div class="row">
+                <div class="col-10 mx-auto col-sm-6 text-center">
+                <h1 class="text-capitalize">our <strong class="banner-title ">store</strong></h1>
+                </div>
+            </div>
+
+            <div class="row">
+                <div class=" col-lg-8 mx-auto d-flex justify-content-around flex-wrap margin-2rem ">
+                <a href="#" class="btn btn-outline-secondary btn-black text-uppercase filter-btn m-2" onclick="filterBtn('all')"> all</a>
```

Comment:
It will work but just good practice. 

It would have been good, If we use id as click param. 
`onclick="flterByType(this)"`  and then we listen `e.target.id ` Hard coded values can change and we will need to modify all the places of filters. In usual scenario we generate filter/ search params through dynamic array. So just good practice. ****